etc/kayobe/inventory/groups

```diff
@@ -24,6 +24,15 @@ seed-hypervisor
 [infra-vms]
 # Empty group to provide declaration of infra-vms group.
 
+[wazuh-manager]
+# Empty group to provide declaration of wazuh-manager group.
+
+[wazuh-agent:children]
+seed
+overcloud
+[wazuh:children]
+wazuh-agent
+wazuh-manager
```

Comment:
I thought you were getting rid of this? I introduced it in some places for generating wazuh-secrets.yml as a group_vars file instead of global extra vars. I don't mind if it's global, but in that case the wazuh group isn't necessary.